src/Equinor.ProCoSys.IPO.WebApi/Properties/launchSettings.json

```diff
@@ -23,6 +23,13 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
       "applicationUrl": "https://localhost:5001;http://localhost:5000"
+    },
+    "Docker": {
```

Comment:
dont know what this is or why it is needed?

Reply:
visual studio autogenerates if you want to run in docker. Its not needed, but it doesn't hurt, and I'm tired of removing it after every run D: